pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -472,7 +472,8 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 	log.Functionf("doUpdateVol(%s) name %s", status.Key(), status.DisplayName)
 
 	// Anything to do?
```

Comment:
now that a better comment it's in place, this one can be removed...

Reply:
Sure